tests/testthat/test-add_stat.tbl_continuous.R

```diff
@@ -0,0 +1,57 @@
+test_that("add_stat() for 'tbl_continuous'", {
+  tt <-
```

Comment:
Where we can (which i think is all places), can we swap `%>%` for the base R pipe?